src/commands/service/pipeline.ts

```diff
@@ -166,16 +162,23 @@ export const installPipeline = async (
     branchFilters: ["master"],
     maximumConcurrentBuilds: 1,
     /* tslint:disable-next-line object-literal-shorthand */
-    pipelineName: pipelineName,
-    repositoryName: repoName,
-    repositoryUrl: repoUrl,
+    pipelineName,
+    repositoryName,
+    repositoryUrl,
     yamlFileBranch: "master",
-    yamlFilePath: path.join(packagesDir, serviceName, "azure-pipelines.yaml") // This may not work if we're using a non-mono repository and azure-pipelines.yaml is in the root directory.
+    yamlFilePath: packagesDir // if a packages dir is supplied, its a mono-repo
```

Comment:
@bnookala note this new logic. Previously it only worked on mono-repos. This logic allows for standard as well

Reply:
👍